libs/akita-ng-effects/src/__tests__/create-effect.util.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { createEffect } from '@datorama/akita-ng-effects';
+import { of } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+describe('Create effect utility', () => {
+  it('should create an effect with effect metadata', () => {
+    const effect = createEffect(() => of('sample'));
+
+    expect(effect).toMatchObject({ isEffect: true });
+    expect(effect).toHaveProperty('name');
+  });
+
+  it('should emit a value upon subscription', (done) => {
+    createEffect(() =>
+      of('sample').pipe(
```

Comment:
Why we need `done` here? It's sync operation.

Reply:
As far as I'm concerned this is async.
The test will finish before the assertion has run.
It results in a false positive once the fakeAsync is removed.